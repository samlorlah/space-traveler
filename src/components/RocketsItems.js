import React from 'react';
import RocketsItemList from './RocketsItemList';

const RocketsItems = () => (
  <div className="rockets">
    <ul>
      <RocketsItemList />
    </ul>
  </div>
);

export default RocketsItems;
