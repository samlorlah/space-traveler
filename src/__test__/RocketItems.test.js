import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import RocketsItems from '../components/RocketsItems';

it('Renders RocketsItems component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <RocketsItems />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
