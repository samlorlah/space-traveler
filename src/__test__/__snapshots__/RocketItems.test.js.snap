// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders RocketsItems component correctly 1`] = `
<div
  className="rockets"
>
  <ul />
</div>
`;
